docker-assets/application.conf

```diff
@@ -0,0 +1,53 @@
+# Timeout: The number of seconds before receives and sends time out.
+Timeout 120
```

Comment:
Are all of these settings really things that *every* app using httpd will want?

At the very least this is making assumptions about ciphers (which people tend to change) and ssl (which I think we are moving to the route and will no longer need in here).

I would think we would provide the entire configuration in our template as a config map, not just parts to be included.